lib/hooks/usePrizePoolContracts.js

```diff
@@ -85,9 +85,15 @@ const _fetchPrizeAndStrategyContractVersions = async (
 
   let prizePoolVersion = {
     address: prizePoolAddress.toLowerCase(),
-    contract: 'StakePrizePool',
+    contract: CONTRACTS.stake,
     version: '3.2.0'
   }
+
+  // Handle special case for this Aave USDT pool released on Polygon:
+  if (prizePoolAddress.toLowerCase() === '0x887e17d791dcb44bfdda3023d26f7a04ca9c7ef4') {
```

Comment:
This should be caught in the regular case if we update the versions object in `current-pool-data`

Reply:
Gotcha